.agents/skills/playwright-best-practices/README.md

```
@@ -0,0 +1,148 @@
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)